src/main/java/com/ait/lienzo/client/core/util/Geometry.java

```diff
@@ -1403,8 +1377,46 @@ public static void getCardinalIntersects(PathPartList path, Point2DArray cardina
                         }
                     }
                     segmentStart = end;
+                    break;
+                }
+                case PathPartEntryJSO.BEZIER_CURVETO_ABSOLUTE:

```

Comment:
Some branches of this Switch is tested but this Case didn't.

Reply:
good catch! done - as now I'm testing a success intersection between curves and line, so it's being called :+1: